Lib/sqlite3/test/backup.py

```diff
@@ -0,0 +1,166 @@
+import os
+import sqlite3 as sqlite
+import unittest
+
+from test import support
+
+
+@unittest.skipIf(sqlite.sqlite_version_info < (3, 6, 11), "Backup API not supported")
+class BackupTests(unittest.TestCase):
+    def setUp(self):
+        cx = self.cx = sqlite.connect(":memory:")
+        cx.execute('CREATE TABLE foo (key INTEGER)')
+        cx.executemany('INSERT INTO foo (key) VALUES (?)', [(3,), (4,)])
+        cx.commit()
+        os.mkdir(support.TESTFN)
+        self.addCleanup(support.rmtree, support.TESTFN)
+        self.temp_counter = 0
+
+    @property
+    def temp_file_name(self):
+        self.temp_counter += 1
+        return os.path.join(support.TESTFN, 'bcktest_%d' % self.temp_counter)
+
+    def tearDown(self):
+        self.cx.close()
+
+    def testBackup(self, bckfn):
+        cx = sqlite.connect(bckfn)
+        result = cx.execute("SELECT key FROM foo ORDER BY key").fetchall()
+        self.assertEqual(result[0][0], 3)
+        self.assertEqual(result[1][0], 4)
+
+    def CheckBadTarget(self):
```

Comment:
Please don't follow the example of the other sqlite tests and use standard test names ``test_foo``.